webapp/src/components/channel_header_dropdown_button/component.tsx

```diff
@@ -93,6 +94,7 @@ const ChannelHeaderDropdownButton = ({
         );
     }
 
+    // TODO: verify isCloudPaid message (asked in channel)
```

Comment:
Has this been verified?

Reply:
Thanks for catching that. Thought I did, but can't find it. Asked again.

Reply:
I removed the redundant subheading. I'll put it back if I hear differently.